src/interfaces/widget/layout.py

```diff
@@ -0,0 +1,55 @@
+"""
+
+    Create a Qt layout wrapper object to arrange child widgets
+    within a widget to ensure that they make good use of the available space.
+
+"""
+
+from PyQt5 import QtWidgets
+
+
+class Layout(QtWidgets.QLayout):
+
+    def __init__(self, widget, **kwargs):
+        super(Layout, self).__init__(widget) #, **kwargs)
+        
+        if not kwargs:
```

Comment:
Unless we want to explicitly check if there `kwargs` is truthy this should probably be structured like so:
```python
if kwargs:
    # Do stuff
```

Reply:
fixed

Reply:
Actually, I think we don't need this check, latter on there's a if condition to detect if we want to call the "set" function. For example:

```python
# Set property if given
name and self.setObjectName(name)
# this line means, if name is not None, then called the setObjectName function
# otherwise, skip this line
```